test/system/250-systemd.bats

```diff
@@ -362,4 +362,25 @@ EOF
     run_podman 1 pod exists test_pod
 }
 
+@test "podman-system-service containers survive service stop" {
+    $SERVICE_NAME=podman-service-$RANDOM
+    URL=tcp://0.0.0.0:$RANDOM
+
+    systemd-run --unit=$SERVICE_NAME bin/podman system service $URL --time=0
```

Comment:
Use `$PODMAN`, not `bin/podman`. And you may want to add `skip_if_remote`. And, there is no other use of `systemd-run` anywhere in the tests, which makes me super uncomfortable.